src/connector/src/sink/doris.rs

```diff
@@ -54,6 +54,8 @@ pub struct DorisCommon {
     pub database: String,
     #[serde(rename = "doris.table")]
     pub table: String,
+    #[serde(rename = "doris.partial_columns")]
```

Comment:
This seems to be a user-facing change. Is there any CI to cover the usage? And also we should have some docs about the user facing change to tell users how to use the options.

Reply:
added Release note